languages/ruby/reference/exercise-concepts/gigasecond.md

```diff
@@ -0,0 +1,12 @@
+== Gigasecond
```

Comment:
```suggestion
## Gigasecond
```
I am not sure where this documentation will end up, so unsure if we should use level 1 header, or level 2.